app/assets/stylesheets/administrate/base/_forms.scss

```diff
@@ -1,78 +1,99 @@
 fieldset {
-  background: lighten($base-border-color, 10);
-  border: $base-border;
-  margin: 0 0 ($base-spacing / 2) 0;
-  padding: $base-spacing;
+  background-color: transparent;
+  border: 0;
+  margin: 0;
+  padding: 0;
 }
 
-input,
-label,
-select {
-  display: block;
-  font-family: $form-font-family;
-  font-size: $form-font-size;
+legend {
+  font-weight: 600;
+  margin-bottom: $small-spacing / 2;
```

Comment:
`$small-spacing / 2` should be written without spaces around the operator: `$small-spacing/2`